docs/description.rst

```diff
@@ -59,6 +59,38 @@ Types of houses:
 | **GMF**: haushaltsähnliche Gewerbebetriebe (household-like business enterprises)
 | **GHD**: Summenlastprofil Gewerbe/Handel/Dienstleistungen (Total load profile Business/Commerce/Services)
 
+Building class:
+
+The parameter ``building_class`` (German: Baualtersklasse) can assume values in range 1-11.
+See page 42 of `this document <https://www.eko-netz.de/files/eko-netz/download/3.5_standardlastprofile_bgw_information_lastprofile.pdf>`_ for guidance on how to determine ``building_class``.
+
+You can either use the building class of 11 for Germany according to table 3 in the linked document, the ``building_class`` assigned to German federal states in table 3:
+
+    ========================= =====================================
+    Federal state                    Building class
+    ========================= =====================================
+    Baden-Württemberg           4
+    Bayern                      5
+    Berlin                      3
+    Brandenburg                 5
+    Bremen                      1
+    Hamburg                     2
+    Hessen                      3
+    Mecklenburg-Vorpommern      6
+    Niedersachsen               3
+    Nordrhein-Westfalen         3
+    Rheinland-Pfalz             4
+    Saarland                    2
+    Sachsen                     4
+    Sachsen-Anhalt              5
+    Schleswig-Holstein          4
+    Thüringen                   5
+    ========================= =====================================
```

Comment:
Is this table a copy of the table in the document? I am unsure if it is such a good idea to reproduce it here. Why not tell that the linked reference contains data on building class for the federal states of Germany (p. 42, Table 3)? Might be better than duplicating the information?

Reply:
Yes, that's right. I have also thought about that. I decided to include the table in the documentation since the linked document is an online source that may not work in the future and because it is the only source which explains `building_class` that I have seen so far.

(It is not an exact duplicate of the table, by the way. I have omitted the information on the distribution of the building age of the federal states.)

Reply:
I agree with @jnnr if the document is available easily. The advantage is that people are forced to look into the concept of building classes and not just use the number of their federal state.

Reply:
I adapted this in commit https://github.com/oemof/demandlib/pull/46/commits/57e8e9273aa56effc21dff26d2240ce84a02861a. I would appreciate a review of the revisions. :)